[2025-03-30T18:44:30.726+0000] {local_task_job_runner.py:120} INFO - ::group::Pre task execution logs
[2025-03-30T18:44:30.741+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: dag_deliverable.create_hive_table manual__2025-03-30T18:44:15.881883+00:00 [queued]>
[2025-03-30T18:44:30.745+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: dag_deliverable.create_hive_table manual__2025-03-30T18:44:15.881883+00:00 [queued]>
[2025-03-30T18:44:30.745+0000] {taskinstance.py:2306} INFO - Starting attempt 1 of 2
[2025-03-30T18:44:30.758+0000] {taskinstance.py:2330} INFO - Executing <Task(PythonOperator): create_hive_table> on 2025-03-30 18:44:15.881883+00:00
[2025-03-30T18:44:30.764+0000] {standard_task_runner.py:64} INFO - Started process 185 to run task
[2025-03-30T18:44:30.766+0000] {standard_task_runner.py:90} INFO - Running: ['airflow', 'tasks', 'run', 'dag_deliverable', 'create_hive_table', 'manual__2025-03-30T18:44:15.881883+00:00', '--job-id', '198', '--raw', '--subdir', 'DAGS_FOLDER/dag_deliverable.py', '--cfg-path', '/tmp/tmpkdwb2gac']
[2025-03-30T18:44:30.767+0000] {standard_task_runner.py:91} INFO - Job 198: Subtask create_hive_table
[2025-03-30T18:44:30.792+0000] {task_command.py:426} INFO - Running <TaskInstance: dag_deliverable.create_hive_table manual__2025-03-30T18:44:15.881883+00:00 [running]> on host airflow
[2025-03-30T18:44:30.843+0000] {taskinstance.py:2648} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='airflow' AIRFLOW_CTX_DAG_ID='dag_deliverable' AIRFLOW_CTX_TASK_ID='create_hive_table' AIRFLOW_CTX_EXECUTION_DATE='2025-03-30T18:44:15.881883+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2025-03-30T18:44:15.881883+00:00'
[2025-03-30T18:44:30.843+0000] {taskinstance.py:430} INFO - ::endgroup::
[2025-03-30T18:44:30.858+0000] {base.py:84} INFO - Using connection ID 'hive_default' for task execution.
[2025-03-30T18:44:30.885+0000] {hive.py:475} INFO - USE `default`
[2025-03-30T18:44:30.900+0000] {hive.py:475} INFO - DROP TABLE IF EXISTS weather.sensor_data
[2025-03-30T18:44:30.959+0000] {hive.py:475} INFO - 
            CREATE EXTERNAL TABLE IF NOT EXISTS weather.sensor_data (
                record_timestamp DATE,
                temperature_salon FLOAT,
                humidity_salon FLOAT,
                air_salon FLOAT,
                temperature_chambre FLOAT,
                humidity_chambre FLOAT,
                air_chambre FLOAT,
                temperature_bureau FLOAT,
                humidity_bureau FLOAT,
                air_bureau FLOAT,
                temperature_exterieur FLOAT,
                humidity_exterieur FLOAT,
                air_exterieur FLOAT
            )
            ROW FORMAT SERDE 'org.apache.hive.hcatalog.data.JsonSerDe'
            STORED AS TEXTFILE
            LOCATION 'hdfs://namenode:9000/topics/sensores/partition=0'
        
[2025-03-30T18:44:31.003+0000] {dag_deliverable.py:134} INFO - âœ… Tabla creada correctamente en Hive
[2025-03-30T18:44:31.004+0000] {python.py:237} INFO - Done. Returned value was: None
[2025-03-30T18:44:31.004+0000] {taskinstance.py:441} INFO - ::group::Post task execution logs
[2025-03-30T18:44:31.007+0000] {taskinstance.py:1206} INFO - Marking task as SUCCESS. dag_id=dag_deliverable, task_id=create_hive_table, run_id=manual__2025-03-30T18:44:15.881883+00:00, execution_date=20250330T184415, start_date=20250330T184430, end_date=20250330T184431
[2025-03-30T18:44:31.059+0000] {local_task_job_runner.py:243} INFO - Task exited with return code 0
[2025-03-30T18:44:31.072+0000] {taskinstance.py:3503} INFO - 1 downstream tasks scheduled from follow-on schedule check
[2025-03-30T18:44:31.078+0000] {local_task_job_runner.py:222} INFO - ::endgroup::
